.eslintrc

```diff
@@ -0,0 +1,5 @@
+extends: fxa/server
+
+rules:
+  handle-callback-err: 0
+  semi: 0
```

Comment:
This rule just disables any checks for semi-colons. We can add more flags to it if we want to fail if we find semicolons or whatever. I turned this rule off here since there are still a couple files which use semicolons.